style/js/shield_defs.js

```diff
@@ -2160,6 +2160,65 @@ export function loadShields(shieldImages) {
     null
   );
 
+  // Sweden
+  shields["SE:RV"] = roundedRectShield(
+    Color.shields.blue,
+    Color.shields.white,
+    Color.shields.white,
+    2,
+    1,
+    null
+  );
+  shields["SE:LV"] = roundedRectShield(
+    Color.shields.blue,
+    Color.shields.white,
+    Color.shields.white,
+    2,
+    1,
+    null
+  );
+  [
+    "AB",
+    "C",
+    "D",
+    "E",
+    "F",
+    "G",
+    "H",
+    "I",
+    "K",
+    "M",
+    "N",
+    "O",
+    "S",
+    "T",
+    "U",
+    "W",
+    "X",
+    "Y",
+    "Z",
+    "AC",
+    "BD",
```

Comment:
If `network=SE:RV` and `network=SE:LV` have identical shields to county routes, `RV` and `LV` belong at the top of this list.

Inline comments describing what these two abbreviations mean should be enough to make it clear that they are not counties.

Reply:
Changes made